Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2.py

```diff
@@ -847,12 +879,22 @@ def get_public_ip_details_command(client: MsGraphClient, args: dict):
     returns:
         Public IP Address Object
     """
-    resource_group = args.get('resource_group')
     address_name = args.get('address_name')
-    response = client.get_public_ip_details(resource_group, address_name)
+    if args.get('resource_group') is not None:
+        resource_group = args.get('resource_group')
+        response = client.get_public_ip_details(resource_group, address_name)
+        address_id = response.get('id')
+    else:
```

Comment:
Seems like resource_group is required so I don't think it would ever be none: https://github.com/demisto/content/blob/17441c2f86ed54a40f8982e1e6363b08e8dfdcb9/Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2.yml#L531

Reply:
Ok your right